README.md

```diff
@@ -41,10 +41,12 @@ Please note that cells with status *deprecated* are not to be used for new desig
 |         Name        |                   Description                                     |    Status    | Superseded By |
 |---------------------|-------------------------------------------------------------------|--------------|---------------|
 | `counter`           | Generic up/down counter with overflow detection                   | active       |               |
+| `delta_counter`     | Up/down counter with variable delta and overflow detection        | active       |               |
 | `generic_LFSR_8bit` | 8-bit linear feedback shift register (LFSR)                       | *deprecated* | `lfsr_8bit`   |
 | `lfsr_8bit`         | 8-bit linear feedback shift register (LFSR)                       | active       |               |
 | `lfsr_16bit`        | 16-bit linear feedback shift register (LFSR)                      | active       |               |
 | `lfsr`              | 4...64-bit parametric Galois LFSR with optional whitening feature | active       |               |
+| `max_counter`       | Up/down counter with variable delta and maximum latch             | active       |               |
```

Comment:
the word "latch" could be misleading in this context. can we replace it with something like "maximum detection"?